services/QuillLMS/app/services/posthog_integration/client.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'posthog'
+
+module PosthogIntegration
+  class Client
+    def track_activity_assignment(user_id:)
```

Comment:
Not making this an ApplicationService because it's likely we'll add additional client methods in the future